setup.py

```diff
@@ -50,7 +50,12 @@
     extras_require=dict(
         tsne=["fitsne"],
         leiden=["leidenalg"],
-        mkl=["mkl"]
+        louvain=["louvain"],
+        demuxEM=["demuxEM"],
+        scanorama=["scanorama"],
+        gprofiler=["gprofiler-official"],
+        torch=["torch"],
```

Comment:
The `torch` list should also include `harmony-pytorch` and `nmf-torch` packages.